core-bundle/src/InsertTag/InsertTagSubscription.php

```diff
@@ -12,6 +12,9 @@
 
 namespace Contao\CoreBundle\InsertTag;
 
+/**
+ * @internal
+ */
```

Comment:
Just a very minor nitpick: theoretically speaking, `@internal` already implies "do not extend", so `final` could be removed now.

Reply:
But the `final` helps ourselfes to not inherit from it ☺️
I’m fine either way.

Reply:
Me too, that‘s why I wrote "theoretically speaking". In my opinion, we should even use `final` more often (see #924).